doc/nvim-tree-lua.txt

```diff
@@ -434,6 +434,7 @@ applying configuration.
         ignore = true,
         show_on_dirs = true,
         show_on_open_dirs = true,
+        disabled_dirs = {},
```

Comment:
Wouldn't it be more consistent with other names to call it something like `disable_for_dirs`?

Reply:
Nice. I'll update.